eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/NavigationHeader.js

```diff
@@ -116,6 +120,13 @@ export class UnwrappedNavigationHeader extends React.Component {
                 <IconText icon={MetadataIcon}>Metadata</IconText>
               </RouteButton>
             </IconListItem>
+            {me.admin && (
+              <IconListItem>
+                <RouteButton variant="tertiary-light" small to={historyUrl}>
+                  <IconText icon={HistoryIcon}>History</IconText>
+                </RouteButton>
+              </IconListItem>
+            )}
           </IconList>
         </QuestionnaireLinks>
         <QuestionnaireContent>
```

Comment:
If putting the other piece of text into a constant is done, `history` and `Metadata` should be too.